extensions/spark/kyuubi-spark-authz/src/main/scala/org/apache/kyuubi/plugin/spark/authz/serde/package.scala

```diff
@@ -82,4 +91,33 @@ package object serde {
       .map(s => s.operationType)
       .getOrElse(QUERY)
   }
+
+  def getExtractor[T <: Extractor](implicit ct: ClassTag[T]): T =
+    getExtractor[T](extractorKey = null)(ct)
```

Comment:
how about calculating the `extractorKey` in this place? to avoid `null` propagation

Reply:
Adopted.